README.md

```diff
@@ -56,9 +56,23 @@ $ make run-console-local
 
 Then visit `http://localhost:9000` to view the console.
 
+## Subscribe to a Package and Channel
+
+Cloud Services can be installed from the catalog by subscribing to a channel in the corresponding package.
+
 **Subscription detail view:**
 ![screenshot_20180628_165240](https://user-images.githubusercontent.com/11700385/42060125-c3cde42c-7af3-11e8-87ec-e5910a554902.png)
 
+NOTE: The operators available in the catalogue by default are that ones which are published in https://operatorhub.io/.
```

Comment:
The default catalogs differ between OpenShift and upstream (OCP has 3!). This is good to mention, but from the perspective of an example, the catalog can be arbitrary. We don't want users to think the default operators are the only ones they can install.